docs/lib/restapi/fragments/android/getting-started/11_amplifyInit.md

```diff
@@ -0,0 +1,49 @@
+To start provisioning api resources in the backend, go to your project directory and **execute the command**:
+
+```bash
+amplify add api
+```
+
+Enter the following when prompted:
+```console
+? Please select from one of the below mentioned services: 
+    `REST`
+? Provide a friendly name for your resource to be used as a label for this category in the project: 
+    `api`
+? Provide a path (e.g., /book/{isbn}): 
```

Comment:
Have you tested the functionality if a customer does setup a greedy path like this?

Reply:
Not yet, no. Added to our list to test.